### DISCUSSION TITLE
Windows Store (UWP) Apps

### DISCUSSION THREAD

**Discussion by rb2k5** [Codectory/AutoActions#10] [General]
Thanks for this terrific application, it something that Windows has been needing for for a while.
Are there any plans to allow users to add Windows Store / UWP apps to the Apps list via the GUI?
Its possible to add them by manually editing the HDRProfile_Settings.xml file with the process name, but to do it in the GUI would be great.

**Comment by Codectory** [comment]
Thanks you! This is something I haven’t thought about. That’s a good point
for a new release!

rb2k5 ***@***.***> schrieb am Do. 11. März 2021 um 12:49:

**Reply by rb2k5** [reply]
That would be great! For now I just create a shortcut to the app and add the process name, eg.
`    <ApplicationItem>
      <DisplayName>Forza Horizon 4</DisplayName>
      <ApplicationName>ForzaHorizon4</ApplicationName>
      <ApplicationFilePath>D:\Apps\Game Tools\HDRProfile\uwp\Forza Horizon 4.lnk</ApplicationFilePath>
      <RestartProcess>false</RestartProcess>
    </ApplicationItem>`
Works a treat!

**Reply by Codectory** [reply]
UWP Apps are now supported. Try 1.5.8

**Reply by rb2k5** [reply]
Crashes for me when I click "Choose UWP App.."
Same in 1.5.10.

**Comment by rb2k5** [comment]
Fixed in 1.6.1.
Thanks.
